File: src/main/java/com/asos/reservationSystem/domain/dto/MeetingDto.java
package com.asos.reservationSystem.domain.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MeetingDto {
    private Long id;
    private LocalDateTime beginning;
    private Integer duration;
    private CourseDto course;
    private UserDto teacher;
    private UserDto student;
}

File: ASOS-Frontend/DTOs/MeetingDto.cs
using System.Text.Json.Serialization;
using DTOs.Account;

namespace DTOs
{
    public class MeetingDto
	{
		[JsonPropertyName("id")]
		public long? Id { get; set; }

		[JsonPropertyName("beginning")]
		public DateTime Beginning { get; set; }

		[JsonPropertyName("duration")]
		public int Duration { get; set; }

		[JsonPropertyName("course")]
		public CourseDto Course { get; set; }

		[JsonPropertyName("teacher")]
		public UserDto? Teacher { get; set; }

		[JsonPropertyName("student")]
		public UserDto? Student { get; set; }
	}
}
